terraform/go.mod

```diff
@@ -4,29 +4,40 @@ go 1.15
 
 replace (
 	github.com/coreos/go-oidc => github.com/gravitational/go-oidc v0.0.3
+	github.com/gravitational/protoc-gen-terraform => /Users/gzigzigzeo/go/src/github.com/gzigzigzeo/protoc-gen-terraform
```

Comment:
This path needs fixing before merging.

Reply:
Yes, but first we need to merge protoc-gen-terraform. I will remove it once we do it.

Reply:
Done

Reply:
@gzigzigzeo in my diff, still shows up as a custom vendor?

Reply:
please cut a tag for protoc-gen-terraform and switch to v tag here